app/manage.py

```diff
@@ -7,4 +7,20 @@
 
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line(sys.argv)
+    args = sys.argv
+
+    execute_from_command_line(args)
+
+    if args[1] == "makemigrations":
```

Comment:
For the sake of understanding, is this intercepting and injecting some commands when this argument is passed, then continuing on elsewhere?